### PR TITLE
JAMES-2334 Introduce RabbitMQ testing tools

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/DockerRabbitMQExtensionTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        connectionFactory = new ConnectionFactory();
+        connectionFactory.setHost(rabbitMQ.getHostIp());
+        connectionFactory.setPort(rabbitMQ.getPort());
+        connectionFactory.setUsername(rabbitMQ.getUsername());
+        connectionFactory.setPassword(rabbitMQ.getPassword());
+    }
+
+    @Test
+    public void containerShouldBeUp() throws Exception {
+        try (Connection connection = connectionFactory.newConnection();) {
```
Comment (chibenwa) [inline server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/DockerRabbitMQExtensionTest.java]: Extra ';'

**Comment by chibenwa** [comment]
test this please

Build result lost.

File: server/queue/queue-rabbitmq/pom.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <name>Apache James :: Server :: Mail Queue :: RabbitMQ</name>
+
+    <dependencies>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```
Comment (mbaechler) [inline server/queue/queue-rabbitmq/pom.xml]: anybody can explain what this is useful for ? If not, please drop it

Comment (chibenwa) [reply server/queue/queue-rabbitmq/pom.xml]: Looks related to some OSGI related black magic...

Comment (mbaechler) [reply server/queue/queue-rabbitmq/pom.xml]: Yes, of course, but we don't have any clue it serves any real purpose

**Comment by aduprat** [comment]
Builds:
👍 :
👎 :

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by mbaechler** [comment]
merged in another PR

**Pull request closed without merging**